gc/verbose/VerboseWriterFileLogging.hpp

```diff
@@ -70,7 +70,7 @@ class MM_VerboseWriterFileLogging : public MM_VerboseWriter
 	virtual bool initialize(MM_EnvironmentBase *env, const char *filename, uintptr_t numFiles, uintptr_t numCycles);
 	virtual void tearDown(MM_EnvironmentBase *env);
 
-	virtual bool openFile(MM_EnvironmentBase *env) = 0;
+	virtual bool openFile(MM_EnvironmentBase *env, bool printInitializedHeader = false) = 0;
```

Comment:
Default param here is false, but true in `VerboseWriterFileLoggingBuffered.hpp` and `VerboseWriterFileLoggingSynchronous.hpp `. Is this intended?